configure.ac

```diff
@@ -251,6 +251,8 @@ AC_CHECK_HEADERS(CoreServices/CoreServices.h, [
 ])
 AC_CHECK_FUNCS(backtrace backtrace_symbols backtrace_symbols_fd)
 
+AC_CHECK_FUNCS_ONCE(sys_siglist)
```

Comment:
`_ONCE` takes care of more of the problems such as what to define, printing output, and has a simpler signature.